src/views/core/view_overlay.cpp

```diff
@@ -0,0 +1,54 @@
+#include "views/view.hpp"
+#include "pointers.hpp"
+#include "gta_util.hpp"
+#include "gui.hpp"
+
+namespace big
+{
+	void view::overlay()
+	{
+		if (!g.window.ingame_overlay.opened || g_gui->is_open())
+            return;
+
+		ImGui::SetNextWindowPos(ImVec2(10.0f, 10.0f), ImGuiCond_Always, ImVec2(0.0f, 0.0f));
+		ImGui::SetNextWindowBgAlpha(0.3f);
+
+		if (ImGui::Begin("overlay", nullptr, ImGuiWindowFlags_NoMove | ImGuiWindowFlags_NoTitleBar | ImGuiWindowFlags_NoResize | ImGuiWindowFlags_AlwaysAutoResize | ImGuiWindowFlags_NoSavedSettings | ImGuiWindowFlags_NoFocusOnAppearing | ImGuiWindowFlags_NoNav))
```

Comment:
looks great so far, only thing i would change is removing the no move flag

Reply:
All right, I will remove the flag later.
Another question: do we want to enable the overlay by default?
Currently it is disabled by default.

Reply:
If it’s ok the left side I recommend we either bump it under the water mark or move it’s position and enable it by default. It’s helpful but also people won’t go out of their way to find something they don’t know about.

Reply:
> All right, I will remove the flag later. Another question: do we want to enable the overlay by default? Currently it is disabled by default.

i would have it enabled

Reply:
I have thought a bit about the positioning.

On the left side we have the problem that the notifications from GTA are rendered there. 
Without the NoMove flag, users could adjust the position themselves, which I personally would also find good.
The negative point is that users would have to bring the overlay into focus first, as it is rendered behind Yim.
Unfortunately there is not much space to click on the overlay..

Reply:
I think we still have an issue of the clipping, my honest suggestion is the watermark isn’t exactly necessary, the unload button can be removed and even made a hotkey as well, yim menu itself can be added into the overlay and the name was never important and this would fix the issue even more so and also would just benefit the overall ui with the addition of the overlay.

Reply:
I would say it would not be the best idea to integrate the menu directly into the overlay. The idea was that the overlay is always visible and always shows the basic info regardless of whether you are actively doing something in the menu or not.

Reply:
Yes I understand that, I’m saying there is a problem with it clipping when the menu is open, that is due to the watermark, my suggestion is to just remove the watermark, change where the unload button is in the menu or add a hornet for it and to add the text yim menu not yim menu itself into the overlay. Now we don’t have a clipping issue and it can be freely adjusted with no clashing. And the watermark doesn’t detract from the overall ui, it’s big and bulky as is.

Reply:
Your idea sounds good so far.
Before I make such "major" changes to the UI, I'll wait for @Yimura and his opinion on this topic.

Reply:
I don't want to make any major changes to the UI as the translation branch is still not merged, I should probably just merge it and then slowly transition other not-yet translated parts later on.

There's also the route that we might just redo the entire UI at some point.

Reply:
Basically, I think the current UI looks good and is also relatively user-friendly.
Unfortunately, there are only minor problems such as with the overlay to integrate it reasonably into the UI.